BSB_lan.ino

```diff
@@ -441,15 +441,36 @@
 #include "BSB_lan_config.h"
 #include "BSB_lan_defs.h"
 
+#if defined(__AVR__)
 #include <avr/pgmspace.h>
-//#include <avr/wdt.h>
+#include <EEPROM.h>
+#endif
 #include <Arduino.h>
+
+#if defined(__AVR__) || defined(__arm__)
 #include <SPI.h>
- //Mega
-#if defined(__AVR__)
+#include <Wire.h>
```

Comment:
This includes Wire.h also for AVRs and may or may not create an instance even though the library is not used. This might consume RAM which is already severely limited on the AVR. Therefore, it should not be included on ARM.

Reply:
Fixed